.github/codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 0.5%
```

Comment:
Also consider to slightly increase this threshold: Qibosoq is not as big as Qibolab, so it is rather normal that a PR affects a larger portion of it.